src/WinGetYamlGenerator/InstallerInfo.cs

```diff
@@ -17,6 +17,10 @@ public enum InstallerKind
         EXE,
         MSI,
         MSIX,
+        INNO,
+        WINX,
```

Comment:
Typo - `WIX`?

Reply:
> Thanks for the contribution! See comments in the file, and also please update the comment in `InstallerKindValueConverter.cs` and also the drop-down in `MainPage.xaml`

got it. I will update the others file later